### PR TITLE
Fix #4797 Add one more choice to copyright violation report 

### LINKED ISSUE #4797: Peer review: Add one more choice to the list
peer review feature:

No FoP in copyright-related reasons for deletion (the second question). This is one of the most common reason for deletion, it should be in the list. Proposed message: "There is no freedom of panorama in the country where this picture was taken, so some installations (even permanently displayed in public spaces) can not be freely photographed and shared."

### PR DESCRIPTION
**Pull request by u7253519**
**Description (required)** 

Added fourth option for peer review copyright violation reason
A follow-up to #5099 which was accidently closed.
Fixes #4797 

**Tests performed (required)**

Tested beta debug build on OPPO A53s with API level 30.

pr2

Need help? See https://support.google.com/android/answer/9075928

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

**Comment by u7253519** [comment]
@nicolas-raoul Can you help me look at this pull request? I accidently closed #5099 so I created this one instead as replacement. Thank you! I`ve also commented on the issue that you talked about in #5099. Please tell me if there`s more changes that has to be made, I will fix it as well, thank you!

File: app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java [hunk 1]
```diff
@@ -192,12 +196,13 @@ public void askReasonAndExecute(Media media,
...
         });
 
+        String[] newReasonListEnglish = reasonListEnglish;
         alert.setPositiveButton(context.getString(R.string.ok), (dialogInterface, i) -> {
 
             String reason = getLocalizedResources(context, Locale.ENGLISH).getString(R.string.delete_helper_ask_alert_set_positive_button_reason) + " ";
 
             for (int j = 0; j < mUserReason.size(); j++) {
-                reason = reason + reasonListEnglish[mUserReason.get(j)];
+                reason = reason + newReasonListEnglish[mUserReason.get(j)];
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java:205]: What error exactly do you get if you don't perform this change?

Comment (u7253519) [reply app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java:205]: It`s this error:
微信图片_20221103175851

**Comment by Ayan-10** [comment]
Could you please resolve the conflicts?

**Comment by shankarpriyank** [comment]
@nicolas-raoul This can also be closed as closed via #5173

**Pull request closed without merging**